Makefile

```diff
@@ -15,22 +15,55 @@ TCTL_SHA := $(shell sh -c "git submodule status -- tctl | cut -c2-40")
 DOCKER ?= docker buildx
 BAKE := IMAGE_TAG=$(IMAGE_TAG) TEMPORAL_SHA=$(TEMPORAL_SHA) TCTL_SHA=$(TCTL_SHA) $(DOCKER) bake
 
+AMD64_BINS ?= build/amd64
+ARM64_BINS ?= build/arm64
+
 ##### Scripts ######
 install: install-submodules
 
 update: update-submodules
 
 install-submodules:
-	@printf $(COLOR) "Installing temporal and tctl submodules..."
-	git submodule update --init $(TEMPORAL_ROOT) $(TCTL_ROOT)
+	@printf $(COLOR) "Installing submodules..."
+	git submodule update --init
 
 update-submodules:
-	@printf $(COLOR) "Updatinging temporal and tctl submodules..."
+	@printf $(COLOR) "Updating submodules..."
 	git submodule update --force --remote $(TEMPORAL_ROOT) $(TCTL_ROOT)
```

Comment:
either remove vars or leave comment alone, since it is specific to those two submodules